docs/API.md

```diff
@@ -1477,23 +1483,39 @@ The `"params"` object sent to `start_domain_test` or `add_batch_job` when the *t
 > TODO: List all possible error codes and describe what they mean enough for clients to know how react to them.
 >
 
-[Add_batch_job]:                #api-method-add_batch_job
+[API add_batch_job]:            #api-method-add_batch_job
+[API add_api_use]:              #api-method-add_api_user
+[API start_domain_test]:        #api-method-start_domain_test
+[API key]:                      #api-key
+[Batch id]:                     #batch-id
+[Client id]:                    #client-id
+[Client version]:               #client-version
+[Domain name]:                  #domain-name
 [DS info]:                      #ds-info
 [Delegation Signer]:            https://datatracker.ietf.org/doc/html/rfc4034#section-5
 [Dot-decimal notation]:         https://en.wikipedia.org/wiki/Dot-decimal_notation
+[IP address]:                   #ip-address
 [ISO 3166-1 alpha-2]:           https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
 [ISO 639-1]:                    https://en.wikipedia.org/wiki/ISO_639-1
 [JSON Pointer]:                 https://datatracker.ietf.org/doc/html/rfc6901
 [JSON-RPC 2.0]:                 https://www.jsonrpc.org/specification
 [LANGUAGE.locale]:              Configuration.md#locale
 [Language tag]:                 #language-tag
 [Name server]:                  #name-server
+[Non-negative integer]:         #non-negative-integer
+[Priority]:                     #priority
 [Privilege levels]:             #privilege-levels
 [Profile name]:                 #profile-name
 [Profile sections]:             Configuration.md#public-profiles-and-private-profiles-sections
+[Progress percentage]:          #progress-percentage
+[Queue]:                        #queue
 [RFC 5952]:                     https://datatracker.ietf.org/doc/html/rfc5952
+[Severity level]:               #severity-level
 [Severity Level Definitions]:   https://github.com/zonemaster/zonemaster/blob/master/docs/specifications/tests/SeverityLevelDefinitions.md
-[Start_domain_test]:            #api-method-start_domain_test
+[Test id]:                      #test-id
+[Test result]:                  #test-result
+[Timestamp]:                    #timestamp
+[Username]:                     #username
 [Validation error data]:        #validation-error-data
 [`RPCAPI.enable_add_api_user`]: Configuration.md#enable_add_api_user
```

Comment:
I suggest having
```
[`RPCAPI.enable_add_api_user`]: Configuration.md#enable_add_api_user
```
in the link list to make them sort correctly, and then
```
[`RPCAPI.enable_add_api_user`][RPCAPI.enable_add_api_user] in the text.
```

There are three such cases.

Reply:
I was working on this at the same time, I chosed to use the following template so that all links related to the configuration are grouped together `[Conf SECTION.key]`. Is this fine for you?